util/conan-package/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake
+import os
+
+channel = os.getenv("CONAN_CHANNEL", "testing")
```

Comment:
Thanks for the feedback! I have implemented the changes and introduced a pull request: https://github.com/g-truc/glm/pull/662